app/experimenter/settings.py

```diff
@@ -379,3 +380,9 @@
 KINTO_BUCKET = config("KINTO_BUCKET")
 KINTO_BUCKET_MAIN = config("KINTO_BUCKET_MAIN")
 KINTO_COLLECTION = config("KINTO_COLLECTION")
+
+
+# Jetstream GCS Bucket data
+DEFAULT_FILE_STORAGE = "storages.backends.gcloud.GoogleCloudStorage"
+GS_PROJECT_ID = "experiments-analysis"
+GS_BUCKET_NAME = "mozanalysis"
```

Comment:
Interesting! Fancy! I guess we don't use any filestorage reading anywhere so setting this as the default doesn't break anything.  If we ever need to read from the filesystem we'll have to update this, but for now this is totally fine, it's so nice that this is so clean and JustWorks™️